R/swagger.R

```diff
@@ -191,5 +346,58 @@ removeNaOrNulls <- function(x) {
   ret <- lapply(x, removeNaOrNulls)
   class(ret) <- class(x)
 
+  # Put example back in
+  if (saveExample) {
+    ret$example <- savedExample
+  }
+
   ret
 }
+
+#' For openapi definition
+#' @noRd
+priorizeProperty <- function(...) {
+  l <- list(...)
+  isnullordefault <- sapply(l, function(x) {isNaOrNull(x) || isTRUE(attributes(x)$default)})
```

Comment:
`sapply` is not a _consistent_ return shape function. Since we are looking for a vector, we can use `vapply`.

```suggestion
  isnullordefault <- vapply(l, function(x) {isNaOrNull(x) || isTRUE(attributes(x)$default)}, logical(1))
```